store/pg_engine.go

```diff
@@ -306,113 +311,181 @@ func migrate(ctx context.Context, d dbdriver.Driver, curVer *semver.Version, cut
 	}
 
 	// Apply migrations
-	versions, err := getMigrationFileVersions()
-	if err != nil {
-		return semver.Version{}, err
-	}
+	retVersion := semver.Version{}
```

Comment:
assign this to *curVer

so at the end of the method, we compare *curVer with retVersion, which will be more readable.

Reply:
Done